### ISSUE TITLE
Xamarin.Android bindings for Sceneform

### ISSUE THREAD

**Issue by sdebruyn** [xamarin/XamarinComponents#475]
Sceneform is an important component of ARCore and enables you to render images in the AR scene. Without it, you can't actually augment the view.

https://developers.google.com/ar/develop/java/sceneform/

> VS bug [#732181](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/732181)

**Comment by newky2k** [comment]
@SamuelDebruyn Let us look into and we'll get back to you

Regards

**Comment by SteHiggins** [comment]
Is it possible to give a rough timescale on this?  It would be incredibly useful.

**Comment by jamboharris** [comment]
Agree with @SteHiggins - this would be a huge win for our project - any hints on timescale? Thanks @Redth @newky2k @SamuelDebruyn

**Comment by ryanlpoconnell** [comment]
This would be useful for our project...  @Redth I'm sad to see this has been moved back into triage.

I hope one of the main benefits of Xamarin doesn't become it's weakness!?... i.e. waiting for the .net bindings on new and exciting libraries mean Xamarin developers / employers are left eating dust in the technology race.

**Comment by Redth** [comment]
We are still working on this :) 

Our automated issue handling process moved it back to triage since it was commented on, no other reason!

**Comment by jamboharris** [comment]
@Redth Awesome!  Excited for some Sceneform goodness soon!

**Comment by simonegarau** [comment]
Any news? Without any timescale on this I'm tempted to just move my project from Xamarin to another solution

**Comment by newky2k** [comment]
@simonegarau  We are currently working on the Google VR binding #517 and then we will look at sceneform

**Comment by jamboharris** [comment]
Thanks for the update @newky2k

**Comment by h-maac** [comment]
Can we get an update on the status of Sceneform @newky2k

**Comment by newky2k** [comment]
@h-maac I'm starting on the binding for this now, just fyi

**Comment by h-maac** [comment]
Thank you very much for the good news @newky2k, I'm pretty sure that you made loads of people happy, including me. Not really sure how long these processes take but I'm looking forward to hear from you. Best!

**Comment by newky2k** [comment]
@h-maac @SamuelDebruyn @jamboharris @SteHiggins @ryanlpoconnell @simonegarau We have pushed out preview1 of the Sceneform for Xamarin.Android binding based on version 1.8.0.  We have the HelloSceneForm working, though we are having issues with applying the materials.

The code is available here -> https://github.com/xamarin/XamarinComponents/tree/master/Android/SceneForm and we have pushed the packages to nuget, just search for Xamarin.Android.SceneForm.

We'd love you feedback and any suggestions(and pointers to fixing the material in the sample) would be great.

Enjoy!
